src/redux/actions/samples/loadSamples.js

```diff
@@ -69,36 +67,15 @@ const toApiV1 = (samples, experimentId) => {
 const loadSamples = (
   experimentId = null, projectUuid = null,
 ) => async (dispatch) => {
-  const { currentApiVersion } = config;
-  let url;
-
-  if (currentApiVersion === api.V1) {
-    url = experimentId ? `/${currentApiVersion}/experiments/${experimentId}/samples`
-      : `/${currentApiVersion}/projects/${projectUuid}/samples`;
-  } else if (currentApiVersion === api.V2) {
-    url = `/${currentApiVersion}/experiments/${experimentId ?? projectUuid}/samples`;
-  }
-
   try {
     dispatch({
       type: SAMPLES_LOADING,
     });
 
+    const url = `/v2/experiments/${experimentId ?? projectUuid}/samples`;
```

Comment:
You can just have the variable `experimentId` and when you use `loadSamples` either pass the actual experimentID or the legacy projectUuid if you think it will be better.

Reply:
expeirmentId and projectUuid are the same thing, cleaned up